<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BackupWidget</class>
 <widget class="QWidget" name="BackupWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>545</width>
    <height>80</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="icon">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="trashButton">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>20</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="configButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>10</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="decryptButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>20</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="backupTarget">
   <property name="geometry">
    <rect>
     <x>91</x>
     <y>50</y>
     <width>249</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Cible</string>
   </property>
  </widget>
  <widget class="QLabel" name="backupName">
   <property name="geometry">
    <rect>
     <x>91</x>
     <y>0</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>28</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Monospace</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>NomSauvegarde</string>
   </property>
  </widget>
  <widget class="QLabel" name="backupSource">
   <property name="geometry">
    <rect>
     <x>91</x>
     <y>30</y>
     <width>249</width>
     <height>16</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="text">
    <string>Source</string>
   </property>
  </widget>
  <widget class="QLabel" name="stateIcon">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>30</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
